src/pages/documentation/adobe-target/release-notes.md

```diff
@@ -6,6 +6,8 @@
 
 * Major version update for [Adobe Target](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-target-android).
 
+Please note that the `locationsDisplayed` and `locationClicked` APIs have been renamed to `displayedLocations` and `clickedLocation` respectively for alignment with iOS.
```

Comment:
can we make this two lines to indicate the alternative API for each, e.g:

Please note that the APIs below have been renamed for alignment with the Adobe Target Mobile SDK for iOS:
- `locationsDisplayed` is now `displayedLocations`
- `locationClicked` is now `clickedLocation`

Reply:
Yup. I'll update either the FAQs or migration guide with the changes for Target, ACC and Places later.